### ISSUE TITLE
Context-less doc error "doc comment contains an invalid Rust code block" in 1.42

### ISSUE THREAD

**Issue by rbtcollins** [rust-lang/rust#71908]
I'm getting this warning:
```
warning: doc comment contains an invalid Rust code block
    |
    = help: mark blocks that do not contain Rust code as text: ```text
```

from `cargo doc --no-deps` - thats the entire thing, no file or line information. I think its related to #56884.
Adding `-v` doesn't help. Nightly produces better diagnostics, but the origin of the error is outside my crate : so thats still somewhat-odd

```
$ cargo +nightly doc --no-deps
warning: doc comment contains an invalid Rust code block
   --> /home/robertc/.cargo/registry/src/github.com-1ecc6299db9ec823/async-std-1.5.0/src/utils.rs:243:42
    |
243 |         $($head)* extension_trait!(@ext (#[doc = $d]) $($tail)*);
    |                                          ^^^^^^^^^^^
    |
    = help: mark blocks that do not contain Rust code as text: ```text

warning: 1 warning emitted
```

### Meta

`rustc --version --verbose`:
```
rustc --version --verbose
rustc 1.42.0 (b8cedc004 2020-03-09)
binary: rustc
commit-hash: b8cedc00407a4c56a3bda1ed605c6fc166655447
commit-date: 2020-03-09
host: x86_64-unknown-linux-gnu
release: 1.42.0
LLVM version: 9.0
```

```
rustc 1.45.0-nightly (fa51f810e 2020-04-29)
binary: rustc
commit-hash: fa51f810e5b9254904b92660e7280b7d6a46f112
commit-date: 2020-04-29
host: x86_64-unknown-linux-gnu
release: 1.45.0-nightly
LLVM version: 9.0
```

This is much better in nightly, but the confusing thing - and the reason I'm filing this bug still is that the reported code is outside my crate.

I'm not even sure where in my crate the origin of the problem is :/.

**Comment by jyn514** [comment]
@rbtcollins what crate was giving you these errors? Can you put together a small-ish example/

**Comment by jackh726** [comment]
Without any source code, I don't think this is possible to reproduce. Given its age, I'm closing.
